index.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'MyEnumerable'
+
```

Comment:
When I test your app I get an error.

![image](https://user-images.githubusercontent.com/38088818/198641567-5500c156-08b7-4799-b924-30e7bc1bd9d8.png)

That's because of the way you `require` your `MyEnumerable` class. When you require a class from another file, your use the filename, not the class name. In your case, you need to require the `my_enumerable` filename instead of the `MyEnumerable` class name.